libpod/runtime.go

```diff
@@ -691,24 +691,15 @@ func (r *Runtime) Info() ([]define.InfoData, error) {
 	}
 	info = append(info, define.InfoData{Type: "store", Data: storeInfo})
 
-	reg, err := sysreg.GetRegistries()
-	if err != nil {
-		return nil, errors.Wrapf(err, "error getting registries")
-	}
 	registries := make(map[string]interface{})
-	registries["search"] = reg
-
-	ireg, err := sysreg.GetInsecureRegistries()
+	regData, err := sysreg.GetRegistriesData()
 	if err != nil {
 		return nil, errors.Wrapf(err, "error getting registries")
 	}
-	registries["insecure"] = ireg
-
-	breg, err := sysreg.GetBlockedRegistries()
-	if err != nil {
-		return nil, errors.Wrapf(err, "error getting registries")
+	for _, reg := range regData {
+		registries[reg.Prefix] = reg
```

Comment:
WIth this in my registries.conf (comments from file removed for brevity):

```
[registries.search]
registries = ['docker.io', 'registry.fedoraproject.org', 'registry.access.redhat.com', 'registry.centos.org', 'quay.io']

[registries.insecure]
registries = ['bad.docker.io']

[registries.block]
registries = ['block.docker.io']

```

I'm not seeing the registries in the search table:

```
registries:
  bad.docker.io:
    Blocked: false
    Insecure: true
    Location: bad.docker.io
    MirrorByDigestOnly: false
    Mirrors: []
    Prefix: bad.docker.io
  block.docker.io:
    Blocked: true
    Insecure: false
    Location: block.docker.io
    MirrorByDigestOnly: false
    Mirrors: []
    Prefix: block.docker.io
```
Should it be showing all?

Reply:
Fixed